localBuildRun.sh

```diff
@@ -9,4 +9,4 @@ if [ $? -eq 1 ]; then
     exit 1
 fi
 
-node build/engine/run.js $*
\ No newline at end of file
+node build/engine/run.js $* --commentErrors
```

Comment:
I think '--commentErrors' shouldn't be hardcoded. We just want to generate an syntax Error file, when an error was found in a new pull request. That's why an extra argument was needed to use it only in the syntax check workflow and not anywhere else. You would have to add this argument inside the yml-file.